src/node/BrowserRunner.ts

```diff
@@ -330,12 +330,10 @@ function pidExists(pid: number): boolean {
   try {
     return process.kill(pid, 0);
   } catch (error) {
-    if (error instanceof Error) {
```

Comment:
I believe that the `instanceof Error` check here is failing in Jest due to https://github.com/facebook/jest/issues/2549. This seems to be timing-dependent in my tests and I've had a hard time reproducing it outside of CI.

Given that `process.kill(...)` is part of the standard library, checking that it throws `Error`s seems unnecessary anyways.